oot.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/bash
+
+
+_self_dir=$(realpath "$(dirname "$0")")
+
+
+function usage {
+	echo 'USAGE:'
+	echo -e "\t$0 [FLAGS] <device> [<device>...]\t\t(Example for akatsuki: $0 akatsuki)"
+	echo
+	echo 'FLAGS:'
+	echo -e "\t$0 -c <compiler>\t\tSelect a compiler to use. Currently supported compilers are gcc, linaro_gcc and clang (defaults to clang)."
+	echo -e "\t$0 -f\t\t\tFlash the kernel after compiling (using fastboot)."
+	exit 1
+}
+
+
+while getopts ':c:f' OPT; do
```

Comment:
Can we add a `-h` here too?

Reply:
Done. It will print the `usage` message.

Reply:
Can `--help` be added as well?

Reply:
Done!